### PR TITLE
Feat:p22_29_tennis_court_test

### PR DESCRIPTION
**Pull request by rapaicd**

### CODE REVIEW

File: Tennis-scheduler/src/test/java/com/tennis/tennisscheduler/repositories/TennisCourtRepositoryTest.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+    @AfterEach
+    void tearDown() {
+        underTest.deleteAll();
+    }
+
+    @Test
+    void itShouldFindObjectById() {
+        //given
+        TennisCourt tennisCourt = new TennisCourt(
```
Comment (nemanjaromanic) [inline Tennis-scheduler/src/test/java/com/tennis/tennisscheduler/repositories/TennisCourtRepositoryTest.java]: You can add Builder from Lombok to TennisCourt, and create object with builder

File: Tennis-scheduler/src/test/java/com/tennis/tennisscheduler/repositories/TennisCourtRepositoryTest.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+@DataJpaTest
+class TennisCourtRepositoryTest {
+
+    @Autowired
+    TennisCourtRepository underTest;
```
Comment (nemanjaromanic) [inline Tennis-scheduler/src/test/java/com/tennis/tennisscheduler/repositories/TennisCourtRepositoryTest.java]: Please rename underTest to something that will say what it is when its called. Now I need to scroll up to check what underTest is

**Comment by nemanjaromanic** [comment]
good joob

File: Tennis-scheduler/src/test/java/com/tennis/tennisscheduler/services/TennisCourtServicesTest.java [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+        TennisCourt tennisCourt2 = new TennisCourt(
+                1,
+                "Teren u Prigrevici",
+                SurfaceType.CLAY,
+                "Jako lep teren u Prigrevici",
+                "slikaZaTerenUPrigrevici",
+                null,
+                null
+        );
+        List<TennisCourt> tennisCourtList = new ArrayList<>();
```
Comment (mirkovicjul) [inline Tennis-scheduler/src/test/java/com/tennis/tennisscheduler/services/TennisCourtServicesTest.java]: You can use List.of(...) here

**Review by nemanjaromanic — APPROVED** [review]

**Pull request merged**